lib/hardware/modules/iob_int_sqrt/iob_int_sqrt.py

```diff
@@ -3,7 +3,210 @@ def setup(py_params_dict):
         "original_name": "iob_int_sqrt",
         "name": "iob_int_sqrt",
         "version": "0.1",
-        "generate_hw": False,
+        "confs": [
+            {
+                "name": "DATA_W",
+                "type": "P",
+                "val": "32",
+                "min": "0",
+                "max": "NA",
+                "descr": "",
+            },
+            {
+                "name": "FRACTIONAL_W",
+                "type": "P",
+                "val": "0",
+                "min": "0",
+                "max": "NA",
+                "descr": "",
+            },
+            {
+                "name": "REAL_W",
+                "type": "P",
+                "val": "DATA_W - FRACTIONAL_W",
+                "min": "0",
+                "max": "NA",
+                "descr": "",
+            },
+            {
+                "name": "SIZE_W",
+                "type": "P",
+                "val": "(REAL_W / 2) + FRACTIONAL_W",
+                "min": "0",
+                "max": "NA",
+                "descr": "",
+            },
+            {
+                "name": "END_COUNT",
+                "type": "F",
+                "val": "(DATA_W + FRACTIONAL_W) >> 1",
+                "min": "0",
+                "max": "NA",
+                "descr": "",
+            },
+            {
+                "name": "COUNT_W",
+                "type": "F",
+                "val": "$clog2(END_COUNT)",
+                "min": "0",
+                "max": "NA",
+                "descr": "",
+            },
+        ],
+        "ports": [
+            {
+                "name": "clk",
+                "descr": "Input port",
+                "signals": [
+                    {
+                        "name": "clk",
+                        "width": 1,
+                        "direction": "input",
+                    },
+                ],
+            },
+            {
+                "name": "rst",
+                "descr": "Input port",
+                "signals": [
+                    {
+                        "name": "rst",
+                        "width": 1,
+                        "direction": "input",
+                    },
+                ],
+            },
```

Comment:
We can group the clock and reset into a single standard port.
(We don't need to specify the signals because they are generated automatically)
```suggestion
                "name": "clk_rst",
                "descr": "Clock and reset",
                "signals": [],
            },
```